api/app/Models/PoolCandidateSearchRequest.php

```diff
@@ -54,6 +54,22 @@ class PoolCandidateSearchRequest extends Model
         'request_status_changed_at' => 'datetime',
     ];
 
+    /**
+     * Boot function for using with PoolCandidateSearchRequest
+     *
+     * @return void
+     */
+    protected static function boot()
+    {
+        /** @var \App\Models\User */
+        $user = Auth::user();
+
+        parent::boot();
+        static::creating(function ($query) use ($user) {
```

Comment:
The parameter name `$query` is a bit strange.  Maybe `$searchRequest` or even just `$model`?

Reply:
Good catch, fixed in [21bdebf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11275/commits/21bdebfc200f4d82b3cc1ab61a59aba94c4b2576)